### DISCUSSION TITLE
[Feature Request] Prepend Query Metadata in SQL Comment

### DISCUSSION THREAD

**Discussion by ndrluis** [apache/superset#23499] [Ideas]
I would like to suggest adding metadata to queries, similar to what is currently available in Re:Dash and Metabase.

**Comment by SamraHanifCareem** [comment]
You can use sql query mutator in superset_config.py to add the relevant metadata

**Comment by mistercrunch** [comment]
https://github.com/apache/superset/blob/master/superset/config.py#L1229-L1249
